### ISSUE TITLE
ODIN support

### ISSUE THREAD

**Issue by andcor02** [ARMmbed/mbed-os-example-wifi#54]
Hi All,

I am aware this example supports the ODIN, but a number of people have flagged how do you actually build the example for the ODIN?? The instructions are very unclear, and they just state using a shield.

Please sort this....

Regards,

Andrea

**Comment by janjongboom** [comment]
@bulislaw @adbridge

**Comment by adbridge** [comment]
@bulislaw over to you :)

**Comment by bulislaw** [comment]
It's actually explicitly mentioned in the README:

> Compile and generate binary
> For example, for GCC:
> mbed compile -t GCC_ARM -m UBLOX_EVK_ODIN_W2

**Comment by bulislaw** [comment]
Improvements to docs are welcomed :)

**Comment by janjongboom** [comment]
@bulislaw Yes, I'm aware, but it's unclear. If we can split this up in two sections, one for boards with built-in wifi and one for using shields, that'd help.

**Comment by yennster** [comment]
Also, for boards using built-in wifi, should the instructions also state whether to include the following in `.mbedignore`?:

```
wifi-x-nucleo-idw01m1/*
esp8266-driver/*
```

**Comment by janjongboom** [comment]
@SeppoTakalo Why are we closing this?! The documentation is unclear and it should be fixed. This issue should be reopened.

**Comment by SeppoTakalo** [comment]
Ublox is mentioned in https://github.com/ARMmbed/mbed-os-example-wifi#supported-hardware

Also in the [Getting started](https://github.com/ARMmbed/mbed-os-example-wifi#getting-started)
 **Configure the WiFi shield to use** section have this clause:
```
For built-in WiFi, ignore the value of wifi-shield
```

And next section `For built-in WiFi, keep the default .mbedignore`

Is this not enough?

Each step have built-in wifi mentioned?

**Comment by janjongboom** [comment]
@andcor02 What's your opinion?

**Comment by andcor02** [comment]
I think this is now fine, its a lot more clear.

Regards

Andrea Corrado
Graduate Engineer
ARM
andrea.corrado@arm.com
Skype: andrea.corrado.arm
Mobile: + 44 7901333702
